tdrs-backend/tdpservice/settings/common.py

```diff
@@ -282,11 +282,10 @@ class Common(Configuration):
 
     # Sessions
     SESSION_ENGINE = "tdpservice.core.custom_session_engine"
-    SIGNED_COOKIE_EXPIRES = 60 * 60 * 24  # 24 hours
+    SIGNED_COOKIE_EXPIRES = 60 * 60 * 12  # 12 hours
     SESSION_COOKIE_HTTPONLY = True
     SESSION_SAVE_EVERY_REQUEST = True
-    #SESSION_EXPIRE_AT_BROWSER_CLOSE = True
-    SESSION_COOKIE_AGE = 10  # 15 minutes
+    SESSION_COOKIE_AGE = 60 * 30  # 30 minutes
```

Comment:
@raftmsohani can confirm that I received the warning for inactivity while logged in via AMS. im gonna wait another 30 minutes, because Im trying to understand the relationship between cookie_age and cookie_expires. max age set to 30 minutes when looking at devtools.